Swiftfin tvOS/Views/VideoPlayer/NativeVideoPlayer.swift

```diff
@@ -57,7 +56,9 @@ struct NativeVideoPlayerView: UIViewControllerRepresentable {
     func updateUIViewController(_ uiViewController: UINativeVideoPlayerViewController, context: Context) {}
 }
 
-class UINativeVideoPlayerViewController: AVPlayerViewController {
+// TODO: Refactor such that this does not subclass AVPlayerViewController. Subclassing is not
+// supported according to the apple docs.
+class UINativeVideoPlayerViewController: AVPlayerViewController, AVPlayerViewControllerDelegate {
```

Comment:
Thank you for your contribution, however this isn't correct. We don't need to subclass from anything for this to work. In `VideoPlayerCoordinator` we create the view like so:

```swift
PreferenceUIHostingControllerView {
    Group {
        if Defaults[.VideoPlayer.videoPlayerType] == .swiftfin {
            VideoPlayer(manager: self.videoPlayerManager)
        } else {
            NativeVideoPlayer(manager: self.videoPlayerManager)
        }
    }
}
.ignoresSafeArea()
```

where the tvOS implementation of `PreferenceUIHostingControllerView` is a view that overrides button presses that we can detect in the Swiftfin video player. As we see above, the native player is wrapped in this view but we don't want it to be. This should be changed to the following, where the native player can handle all of its own button presses:

```swift
if Defaults[.VideoPlayer.videoPlayerType] == .swiftfin {
    PreferenceUIHostingControllerView {
        VideoPlayer(manager: self.videoPlayerManager)
    }
    .ignoresSafeArea()
} else {
    NativeVideoPlayer(manager: self.videoPlayerManager)
}
```

Luckily I was going to work on PRs and other work today, but please don't ping me in the future for reviews. I do see everything, but only get to stuff when I can.

Reply:
>  please don't ping me in the future for reviews

Apologies.  It wasn't clear how to get a review and noticed that in an accepted PR someone pinged a maintainer.  Won't do in the future.

Incorporated these suggestions.  PTAL.